Core/Configuration/IrcConfig.cs

```diff
@@ -18,6 +18,9 @@ public sealed class IrcConfig : ConfigBase
         public string Username { get; private set; } = "justinfan27365461784";
         public string Password { get; private set; } = "oauth:mysecret";
 
+        // TODO this configurations should probably be in the database instead
+        public IImmutableList<string> OperatorNames { get; private set; } = ImmutableList<string>.Empty;
```

Comment:
Haven't figured out where and how exactly I'd put that, and if putting it in the database even if a desirable thing. E.g. if I get a database dump from stream I wouldn't want that to interfere with my development configuration

Reply:
database would allow the list to change dynamically, but it might also slow stuff down if we wanted to get a list of operators

Reply:
I wouldn't worry about performance. Might be a good idea to add some sort of generic role system that things like "trusted" can use too. Anyway, can I do this later/in a separate branch?

Reply:
yeah that sounds fine to me